csm/cli/support_bundle/csm_bundle_generate.py

```diff
@@ -139,25 +139,25 @@ def generate_bundle(args):
         os.makedirs(target_path,exist_ok=True)
 
         csm_size_filtered_logs_dir = os.path.join(const.CSM_SETUP_LOG_DIR,
-            f"{const.CSM_COMPONENT_NAME}_logs_size")
+            f"{const.CSM_COMPONENT_NAME}_logs_")
         GenerateCsmBundle.__clear_tmp_files(csm_size_filtered_logs_dir)
         os.makedirs(csm_size_filtered_logs_dir,exist_ok=True)
-        csm_size_time_filtered_logs_dir = os.path.join(const.CSM_SETUP_LOG_DIR,
-            f"{const.CSM_COMPONENT_NAME}_logs_size_time")
-        GenerateCsmBundle.__clear_tmp_files(csm_size_time_filtered_logs_dir)
-        os.makedirs(csm_size_time_filtered_logs_dir,exist_ok=True)
-        FilterLog.limit_size(csm_log_path, csm_size_filtered_logs_dir,
-            GenerateCsmBundle.size_limit, const.CSM_COMPONENT_NAME)
+        
+        csm_time_filtered_logs_dir = os.path.join(const.CSM_SETUP_LOG_DIR,
+            f"{const.CSM_COMPONENT_NAME}_logs_")
```

Comment:
does csm_size_filtered_logs_dir and csm_time_filtered_logs_dir expected to have same path i.e. "/tmp/csm_logs_" ?

Reply:
I have checked the code for utils.. It is better to keep to different path.. I will modify the code